versioning/versions/versions.go

```diff
@@ -26,6 +26,7 @@ type Version struct {
 	Tags         []string
 	From         string
 	Cmd          string
+	TemplateArgs map[string]string
```

Comment:
Or if you want to keep `TemplateArgs`, please add `` `yaml:"templateArgs"` `` annotation (similar to `ExcludeTests`) so that in the yaml file, we can use `templateArgs` instead of `templateargs`.